SDK/AppCenterCrashes/Microsoft.AppCenter.Crashes.Windows.Shared/Utils/ErrorLogHelper.cs

```diff
@@ -141,6 +141,25 @@ public static FileInfo GetStoredErrorLogFile(Guid errorId)
             return GetStoredFile(errorId, ErrorLogFileExtension);
         }
 
+        /// <summary>
+        /// Reads error log file from the given file.
+        /// </summary>
+        /// <param name="fileInfo">The file that contains error log.</param>
+        /// <returns>An error log instance or null if the file doesn't contain error log.</returns>
+        public static ManagedErrorLog ReadErrorLogFile(FileInfo fileInfo)
+        {
+            try
+            {
+                var errorLogString = File.ReadAllText(fileInfo.FullName);
+                return (ManagedErrorLog)LogSerializer.DeserializeLog(errorLogString);
+            }
+            catch (System.Exception e)
+            {
+                AppCenterLog.Error(Crashes.LogTag, $"Encountered an unexpected error while reading an error log file: {fileInfo.Name}", e);
```

Comment:
Should the file be deleted in this case?

Reply:
If we can't do anything with it I suppose there's no reason to keep it, should we log any of the file info other than the name as verbose?

Reply:
This method reads files as the method name says it. I think deleting file is a business logic of Crashes class so it should be removed there.